### PR TITLE
shopinvader_api_sale: explicit filtering on partner

### PR DESCRIPTION
**Pull request by sebastienbeau**

### CODE REVIEW

File: shopinvader_api_sale/routers/sales.py [hunk 1]
```diff
@@ -39,30 +45,40 @@ def search(
...
     )
 
 
 class ShopinvaderApiSaleSalesRouterHelper(models.AbstractModel):
     _name = "shopinvader_api_sale.sales_router.helper"
     _description = "Shopinvader Api Sale Service Helper"
 
-    @property
-    def filtered_model(self):
+    def _get_domain_adapter(self, partner):
```
Comment (lmignon) [inline shopinvader_api_sale/routers/sales.py]: @sebastienbeau To ease extension, partner_id should be a property of ShopinvaderApiSaleSalesRouterHelper

```

class ShopinvaderApiSaleSalesRouterHelper(models.AbstractModel):

    partner = fields.One2many("res.partner")



# To use-it:

env["shopinvader_api_sale.sales_router.helper"]._new({"partner": partner}).get(sale_id)

```

**Comment by lmignon** [comment]
@sebastienbeau I'm looking for the tests

File: shopinvader_api_sale/routers/sales.py (hunk 1 above)
Comment (sebastienbeau) [reply shopinvader_api_sale/routers/sales.py]: I was hesitating with this solution ;) I will change it

**Comment by sebastienbeau** [comment]
@lmignon it's should be ready

**Pull request merged**